sormas-api/src/main/java/de/symeda/sormas/api/caze/CaseFacade.java

```diff
@@ -61,9 +61,9 @@
 
 	Page<CaseIndexDto> getIndexPage(CaseCriteria caseCriteria, Integer first, Integer max, List<SortProperty> sortProperties);
 
-	Page<CaseIndexDetailedDto> getIndexDetailedPage(CaseCriteria caseCriteria, Integer first, Integer max, List<SortProperty> sortProperties);
+	Page<CaseIndexDetailedDto> getIndexDetailedPage(CaseCriteria caseCriteria, Integer offset, Integer max, List<SortProperty> sortProperties);
```

Comment:
Can you rename `page` to `offset` in CaseResource as well for the two index list methods?